spec/loophp/collection/CollectionSpec.php

```diff
@@ -3756,39 +3729,26 @@ public function it_can_unpair(): void
 
     public function it_can_until(): void
     {
-        $collatz = static function (int $initial = 1): int {
+        $collatz = static function (int $initial = 1): array {
             return 0 === $initial % 2 ?
-                $initial / 2 :
-                $initial * 3 + 1;
+                [$initial / 2] :
+                [$initial * 3 + 1];
         };
 
         $this::unfold($collatz, 10)
+            ->unwrap()
+            ->normalize()
```

Comment:
I think we can get rid of `normalize` here as well.

Reply:
you're right although not without updating the test. I made the update now